mochify/lib/load-config.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const { cosmiconfig } = require('cosmiconfig');
+
+exports.loadConfig = loadConfig;
+
+async function loadConfig(config) {
+  const explorer = cosmiconfig('mochify', {
+    searchPlaces: [
+      'package.json',
+      '.mochifyrc',
+      '.mochifyrc.json',
+      '.mochifyrc.yaml',
+      '.mochifyrc.yml',
+      '.mochifyrc.js',
+      '.mochifyrc.cjs',
+      'mochify.config.js',
+      'mochify.config.cjs'
+    ]
+  });
+
+  const config_path = config.config;
+  const config_from_file = config_path
+    ? await explorer.load(resolveConfig(config_path))
+    : await explorer.search();
+
+  if (config_from_file) {
+    Object.assign(config, config_from_file.config, config);
+  }
+}
+
+function resolveConfig(config_path) {
```

Comment:
Is this function really needed? If I understand the cosmiconfig docs correctly calling load on an inexistent file will fail anyways, so Mochify wouldn't even need to try to work around this. Or is this doing something else that I don't yet understand?

Reply:
I shamelessly copied the code from `lint-staged`. I removed this for now as I don't see that it's needed either.